jdisc_http_service/src/test/java/com/yahoo/container/logging/LogFileHandlerTestCase.java

```diff
@@ -184,4 +177,45 @@ public String format(LogRecord r) {
         h.shutdown();
     }
 
+    @Test
+    public void testcompression_zstd() throws InterruptedException, IOException {
```

Comment:
Perhaps these to tests could have refactored since 95% of the code is common.

Reply:
Agree, I will fix in this PR

Reply:
Fixed